src/pages/wordweekPage/WordWeek0.jsx

```diff
@@ -0,0 +1,78 @@
+import ExampleToggle from './ExampleToggle.jsx';
+
+const WordWeek0 = () => {
+    const [visibleExamples, setVisibleExamples] = useState([]);
+
+    const handleExampleToggle = wordId => {
+        const updatedVisibleExamples = ExampleToggle(visibleExamples, wordId);
+        setVisibleExamples(updatedVisibleExamples);
+    };
+
+    const isVisible = wordId => {
+        return visibleExamples.includes(wordId);
+    };
+
+    return (
+        <div className="wordweek0-container">
+            <Header isNavigationBar={true} viewName="ViewName" />
+            <div className="content">
+                {WordData.map((content, index) => (
+                    <React.Fragment key={content.wordId}>
+                        <div
+                            key={content.wordId}
+                            className="item"
+                            style={{ top: `${index * 135}px` }}
+                        >
+                            <div className="item-state">
+                                <div className="item-info">
+                                    <div className="item-standard">
+                                        <div className="item-name">
+                                            {content.wordKr}
+                                        </div>
+                                        <div className="pronunciation">
+                                            {content.pronunciation}
+                                        </div>
+                                    </div>
+                                    <div className="item-type">
+                                        {content.meaning}
+                                    </div>
+                                </div>
+                                <div className="example">
+                                    <div
+                                        className="item-example"
+                                        onClick={() =>
+                                            handleExampleToggle(content.wordId)
+                                        }
+                                    >
+                                        {isVisible(content.wordId)
+                                            ? 'Hide'
+                                            : 'About Word'}
+                                    </div>
+                                </div>
+                            </div>
+                            <div className="bookmark-icon">
+                                <BookmarkIcon wordId={content.wordId} />
+                            </div>
+                        </div>
+                        {isVisible(content.wordId) && (
+                            <ExampleSentence
+                                wordId={content.wordId}
+                                exampleSentences={content.exampleSentence}
+                            />
+                        )}
+                        <img className="line-week" src={Line} alt="Line" />
```

Comment:
component 폴더 내에 Divider.jsx 컴포넌트가 존재합니다! 
```suggestion
                        <Divier />
```
```jsx
import '../../components/Divider';
```
임포트해서 사용해주시면 될 것 같습니다